src/less/badge/ds6/badge.less

```diff
@@ -0,0 +1,12 @@
+@import '../base/badge.less';
+@import "../../less/ds6/variables.less";
+@import "../../less/ds6/mixins.less";
+
+.badge {
+    background-color: @ds6-color-badge-background;
+    color: @ds6-color-badge-text;
+
+    &-container .badge {
+        border: 2px solid @ds6-color-badge-text;
```

Comment:
Let's add `border: 2px solid;` to the `/badge/base/badge.less` file, and just do `border-color: @ds6-color-badge-text;` here. That makes it explicit we're only setting the color here, and not the entire border rule.